app/views/layouts/application.html.erb

```diff
@@ -56,6 +56,6 @@
   <%= yield %>
 <% end %>
 
-<% content_for(:footer_version) { CURRENT_RELEASE_SHA } %>
+<% content_for :footer_version, ENV.fetch("SENTRY_RELEASE", "null") %>
```

Comment:
What do you think of only showing a short SHA (first 10 characters) like it was before?

```suggestion
<% content_for :footer_version, ENV.fetch("SENTRY_RELEASE", "null")[0..18] %>
```